src/components/utils/dom.ts

```diff
@@ -11,6 +11,13 @@ export function nodeListToArray(domNodeList): Element[] {
   }
 }
 
+export function arraymove(arr, fromIndex, toIndex) {
```

Comment:
fixme: rename to `arrayMove`?

Is there a lodash function we can use for this? Does this belong in a dom util if its just for arrays?

Reply:
might be a better home for it. It's too simple to bother loading lodash for imo. I'm honestly surprised there isn't a native DOM equivalent.

renamed.

Reply:
Can you move it into an array.ts file?